codespell_lib/data/dictionary.txt

```diff
@@ -20470,6 +20474,7 @@ posponed->postponed
 posption->position
 possabilites->possibilities
 possabilities->possibilities
+possability->possibility
 possabilties->possibilities
 possabily->possibly
 possable->possible
```

Comment:
I wonder if `possable` should also suggest `posable` ? Might come up when e.g. writing about 3D animation software?
Although ironically, it seems a few occurrences of `posable` on grep.app are actually typos for `possible` :laughing:

Reply:
(as always, feel free to ignore me if you think this is too unlikely)

Reply:
i'm not sure about this one. But i found something, that I put in https://github.com/codespell-project/codespell/pull/1747